source/features/bugs-tab.tsx

```diff
@@ -24,14 +29,49 @@ async function highlightBugsTabOnIssuePage(): Promise<void | false> {
 	issuesTab.removeAttribute('aria-current');
 }
 
-const countBugs = cache.function(async (): Promise<number> => {
-	const {search} = await api.v4(`
-		search(type: ISSUE, query: "label:bug is:open is:issue repo:${getRepo()!.nameWithOwner}") {
-			issueCount
+async function countBugsWithUnknownLabel(): Promise<number> {
+	const {repository} = await api.v4(`
+		repository() {
+			labels(query: "bug", first: 10) {
+				nodes {
+					name
+					issues(states: OPEN) {
+						totalCount
+					}
+				}
+			}
+		}
+	`);
+
+	const label: AnyObject | undefined = repository.labels.nodes
+		.find((label: AnyObject) => isBugLabel(label?.name));
```

Comment:
👍 

But I assume `label` exists, right? I don’t think `nodes` would be an array of `undefined`s
```suggestion
		.find((label: AnyObject) => isBugLabel(label.name));
```

Other than this the commit look good, it's missing the second part (which would solve this TODO):

https://github.com/sindresorhus/refined-github/blob/ae247fb90a4c4b20524d76c823e97af3881326fb/source/github-helpers/search-query.ts#L88

The regex to fix is this, but it might be a little difficult 😅 

https://github.com/sindresorhus/refined-github/blob/ae247fb90a4c4b20524d76c823e97af3881326fb/source/github-helpers/search-query.ts#L5-L8

Reply:
> But I assume `label` exists, right? I don’t think `nodes` would be an array of `undefined`s

No what if there are no bug labels?
it will return
```graphQL
    "repository": {
      "labels": {
        "nodes": []
      },
```

Reply:
> Other than this the commit look good, it's missing the second part (which would solve this TODO):
> 
> https://github.com/sindresorhus/refined-github/blob/ae247fb90a4c4b20524d76c823e97af3881326fb/source/github-helpers/search-query.ts#L88
> 
> The regex to fix is this, but it might be a little difficult 😅

Lost you here

Reply:
> No what if there are no bug labels?

That's an empty array, it means it will never call the `.find` callback:

```js
[].find(console.log)
```

Reply:
> Lost you here

I'm talking about this: > adding support for quoted properties in SearchQuery

i.e. fixing that TODO comment

Reply:
In short this does not work now:

```js
new SearchQuery('label:"my name"').includes('label:"my name"')
```


because:

```js
'label:"my name"'
// is split into
['label:"my', 'name"'`]
// so
['label:"my', 'name"'`].includes('label:"my name"') === false

Reply:
This might be the solution, but we also need to support quote-less values: https://stackoverflow.com/q/16261635/288906

Reply:
> is split into
> 
> ```js
> ['label:"my', 'name"'`]
> ```

Why do you say so?

https://regex101.com/r/FBLn1f/1 looks fine to me.

I also tested the feature and its working perfectly.

The only issue I see is with unquoted searches `label:my name`. But in theory that should never happen except on a manual search. But anyways GitHub search will not work if the value is not quoted.

Reply:
I say so because the comment in the code says so but it looks like https://github.com/sindresorhus/refined-github/pull/3837 fixed it 🎉 

so perhaps this PR is ready once you delete that comment 

https://github.com/sindresorhus/refined-github/blob/ae247fb90a4c4b20524d76c823e97af3881326fb/source/github-helpers/search-query.ts#L88

Reply:
Should I add a test?